packages/core/src/Modifiers.ts

```diff
@@ -306,6 +312,20 @@ export class Modifiers extends EventMixin {
     map<T>(callbackfn: (value: Modifier, index: number, array: Modifier[]) => T): T[] {
         return this._contents?.map(callbackfn) || [];
     }
+    /**
+     * Iterate through all modifiers.
+     */
+    [Symbol.iterator](): Iterator<Modifier> {
```

Comment:
Et du coup avec ça tu peux faire `for (const modifier of modifiers)` avec `modifiers instanceof Modifiers` ? :+1: